site/frontend/src/pages/compare/prefs.ts

```diff
@@ -0,0 +1,10 @@
+import {createStoredValue} from "../../storage";
+
+export const PREF_FILTER_OPENED = createStoredValue(
```

Comment:
Nit: 

```suggestion
export const PREF_FILTERS_OPENED = createStoredValue(
```

plural, like aggregations, since there are a bunch of filters.